app/models/world_location.rb

```diff
@@ -35,7 +35,7 @@ class WorldLocation < ApplicationRecord
   searchable title: :title,
              link: :search_link,
              description: :search_description,
-             only: :active,
+             only: :active_and_international_delegation,
```

Comment:
This is maybe just me, but would this be better as `active_international_delegations`?  At first glance it looks like you're sending active world locations and international delegation world locations.